package.json

```diff
@@ -58,12 +58,13 @@
     "report-dir": "cypress-coverage"
   },
   "dependencies": {
-    "@capacitor-community/bluetooth-le": "3.0.0",
-    "@capacitor/android": "5.0.4",
-    "@capacitor/app": "5.0.5",
-    "@capacitor/core": "5.0.4",
-    "@capacitor/ios": "5.0.4",
+    "@capacitor-community/bluetooth-le": "6.0.0",
+    "@capacitor/android": "^6.0.0",
+    "@capacitor/app": "^6.0.0",
+    "@capacitor/core": "^6.0.0",
+    "@capacitor/ios": "^6.0.0",
     "@ethereumjs/util": "9.0.3",
+    "@ledgerhq/hw-transport": "^6.30.1",
```

Comment:
How come this dependency was added?

Reply:
the reason is that `npx cap migrate` internally calls `yarn upgrade` https://github.com/ionic-team/capacitor/blob/main/cli/src/tasks/migrate.ts#L428C45-L428C52. Adding this dep is not the end of a world because we are importing that dependency in our code base, but I don't want to use `yarn upgrade` anywhere. Will do migration "more manually".